src/assets/css/chart.css

```diff
@@ -0,0 +1,72 @@
+.charts {
```

Comment:
Shouldn't the chart css go into it's component folder and be included via `styleUrls: []`?